Tests/base/KVC/safe_caching.m

```diff
@@ -0,0 +1,66 @@
+#import <Foundation/NSKeyValueCoding.h>
+#import <Foundation/NSValue.h>
+#import <objc/runtime.h>
+
+#import "Testing.h"
+
+// Caching is only available for libobjc2
+#if defined(__OBJC2__)
```

Comment:
I don't understand why we cannot run this test with an old compiler. There isn't much need for it, but without caching this should just work out of the box.